pyof/v0x04/controller2switch/role_reply.py

```diff
@@ -16,14 +16,16 @@
 class RoleReply(RoleBaseMessage):
     """RoleReply Message."""
 
-    def __init__(self, xid=None, role=None, generation_id=None):
+    def __init__(self, xid=None, role=ControllerRole.OFPCR_ROLE_NOCHANGE,
+                 generation_id=0xffffffffffffffff):
```

Comment:
`0xffffffffffffffff` Same here, I am not sure what this number represents, but it may be an already defined constant somewhere. =)

Reply:
This does not exist in 1.0, and is specified in a spec paragraph. I can, as stated above, create a constant for it in `pyof.v0x04.common.constants`.

Reply:
[same from above answer]